src/tejashree/Assignment29.java

```diff
@@ -0,0 +1,24 @@
+package tejashree;
+
+public class Assignment29 {
+	void findSecondMaxInArray(int[] arr) {
+		int max=arr[0];
+		int secondMax=0;		
+		for(int index=0;index<arr.length;index++) {
+			if(arr[index]>max) {
+				secondMax=max;
+				max=arr[index];
+			}
+			else if(arr[index]>secondMax) {
```

Comment:
you should also check
else if (input[index] > secondMax && input[index] != max)